### ISSUE TITLE
recommended that you use use_reentrant=False

### ISSUE THREAD

**Issue by sandyis** [FartyPants/Training_PRO#14]
I see below warning in logs when running a LoRA training , can this be ignored ?

`/text-generation-webui-main/installer_files/env/lib/python3.11/site-packages/torch/utils/checkpoint.py:429: UserWarning: torch.utils.checkpoint: please pass in use_reentrant=True or use_reentrant=False explicitly. The default value of use_reentrant will be updated to be False in the future. To maintain current behavior, pass use_reentrant=True. It is recommended that you use use_reentrant=False. Refer to docs for more details on the differences between the two variants.
  warnings.warn(
`
![image](https://github.com/FartyPants/Training_PRO/assets/24884891/07cbd606-4564-4cca-9fcc-cde1fde4796d)

settings used 
Text: (Service_API.txt) has 1604 blocks (Block Size 256 tokens)
[Batch Size: 7, Epochs: 3.0, Gradient Accumulation: 1]
Total number of steps: 690
Steps per each Epoch: 230
Suggestions:
Checkpoints: Save every 69 - 138 steps (Current: 192)
Warmup steps: 69 (Current: 96)

**Comment by FartyPants** [comment]
It could be ignored. I thought I disabled the warning... have to check
